src/Components/firestore-access.js

```diff
@@ -87,120 +87,128 @@ export const initDates = async (db, symbols, Rounds) => {
   let yearDiff = year - (IPOyearMax+1);
   let maximum_period = Math.floor(((yearDiff * 12)  - min_window_size) / Rounds);
   let random_period =  Math.floor((Math.random()  * (maximum_period - min_window_size))+min_window_size);
-  let startDate = new Date(IPOyearMax+1,1,1);
-  let endDate =  new Date(IPOyearMax+1,1+random_period,1);
+  let startDate = new Date(IPOyearMax + 1, 1, 1);
+  let endDate =  new Date(IPOyearMax + 1, 1 + random_period, 1);
   let rand_startDate =  randomDate(startDate,endDate);
   let dates = [];
   let curr_date = rand_startDate;
+
   for(var i = 0; i < Rounds; i++) {
     dates.push(curr_date.toISOString().substring(0, 10));
     curr_date = new Date(curr_date.setMonth(curr_date.getMonth()+random_period));
   }
+
   const datesD ={
     "dates" : dates,
     "period": random_period
   }
-  return datesD
+  return datesD;
 }
 
-export const initSymbols = async(db,Industry,Sector,NumOfSymbols) =>{
```

Comment:
please add some comments for each export function and explain what does this function do

Reply:
just added!